### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by uggla**
- Put in place sphinx mechanism to build documentation.
  Most of the documentation is generated from python docstrings.
- redfish-client.py usage is a bit modified to display a decent
  documentation.
- A link rfclient.py to redfish-client.py was created to allow sphinx
  parsing. The '-' char is not allowed for module and so sphinx autodoc
  module.
- Note : Copyright may need to be reviewed as well as main.py.

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -1,75 +1,332 @@
...
 project = u'python-redfish'
-copyright = u'2013, OpenStack Foundation'
+copyright = u'2016, Bruno Cornec, Vincent Misson, René Ribaud'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = '0.4'
```
Comment (bcornec) [inline doc/source/conf.py:58]: Would be good to use PBVER here and to add that file to the files processed by pb in python-redfish.pb.

File: doc/source/conf.py [hunk 2]
```diff
@@ -1,75 +1,332 @@
...
+# The full version, including alpha/beta/rc tags.
+release = '0.4'
+
```
Comment (bcornec) [inline doc/source/conf.py:61]: Similarly...

**Comment by bcornec** [comment]
The dep on shpinx should also be reported in the rpm/deb dependencies. I'll add a patch for that later on.

**Comment by uggla** [comment]
Hello Bruno,

Thanks for merging so early in the morning ! :)
Haha, I thought about the PBVER stuff right after clicking the validate button of the pull request.
I will fix that in a next commit that should come up very soon.

**Pull request merged**